watch/views.py

```
@@ -0,0 +1,46 @@
+
+from watch.forms import TargetForm
+from watch.models import Observation, Target
+
+
+class HomeView(TemplateView):
+    """Dashboard landing page."""
+
+    template_name = "watch/home.html"
+
+
+def healthz(request: HttpRequest) -> JsonResponse:
```

Ruff (0.15.20)

[warning] Unused function argument: `request`

(ARG001)